.drone.yml

```diff
@@ -131,25 +130,22 @@ build:
       repo: vmware/vic
       event: [ push, tag ]
       branch: [ master, releases/*, refs/tags/* ]
-      success: true
+      status: success
 
-notify:
-  slack:
-    webhook_url: $$SLACK_URL
-    channel: mwilliamson-staff
+  notify-slack:
+    image: plugins/slack
+    webhook: ${SLACK_URL}
     username: drone
     template: >
-      build https://ci.vmware.run/vmware/vic/{{ build.number }} finished with a {{ build.status }} status, find the logs here: https://console.cloud.google.com/m/cloudstorage/b/vic-ci-logs/o/integration_logs_{{ build.number }}_{{ build.commit }}.zip?authuser=1
+      build https://ci.vcna.io/vmware/vic/{{ build.number }} finished with a {{ build.status }} status, find the logs here: https://console.cloud.google.com/m/cloudstorage/b/vic-ci-logs/o/integration_logs_{{ build.number }}_{{ build.commit }}.zip?authuser=1
     when:
       repo: vmware/vic
-      event: [ push, tag ]
       branch: [ master, releases/*, refs/tags/* ]
-      failure: true
+      status: failure
 
-publish:
-  gcs:
-    auth_key: >
-      $$SERVICE_ACCOUNT_KEY
+  publish-gcs-builds:
+    image: maplain/drone-gcs:latest
```

Comment:
https://github.com/maplain/drone-gcs 

The current drone-gcs plugin does not work against 0.5, @maplain wrote a new plugin to work with 0.5